docs/content/get-started/upload-launch.md

```diff
@@ -36,52 +35,88 @@ Your `spatialos.json` should look like this:
+To do this:
 
+* First, in the **Assembly** section of the Deployments window, enter the **Assembly Name** field. The assembly name is a string you choose made up of alphanumeric characters, `_`, `.`, and `-`; for example `my_assembly`. (As shown in the example below.)
+* Now select **Upload Assembly** to start the upload.
 
-```
-spatial cloud upload my_assembly
-```
+![]({{assetRoot}}assets/deployment-window.png)
+<br/> _Example Deployments window showing the Assembly section completed_
 
 <br/>
-<br/>
-**It’s finished uploading when:** You see an upload report printed in your terminal output, for example:
+
+**It’s finished uploading when:** You see an upload report printed in your Unity console, for example:
+
 ```
-Upload report:
-- 5 artifacts uploaded (4 successful, 1 skipped, 0 failed)
+Uploaded assembly my_assembly to project unity_gdk successfully.
 ```
+
+> **TIP:** Depending on your network speed, the upload may take a little while (1-10 minutes) to complete.
+
+
+#### 3. Launch a cloud deployment
+
+The next step is to [start a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. When you launch a cloud deployment you have to provide four parameters:
+
+* **Assembly Name** - this identifies the worker assemblies to use. 
+* **Deployment Name** - this labels your deployment in the SpatialOS Console.
+* **Snapshot File** - this specifies which snapshot file the deployment uses. A snapshot declares the state of the world as the deployment starts.
+* **Launch Config File** - this is the deployment's launch configuration `.json` file which declares the world and load balancing configuration.
+
 <br/>
-Based on your network speed, this may take a little while (1-10 minutes) to complete.
+To launch your cloud deployment:
 
+First enter the parameters in the Deployment Launcher section of the Deployments window. Complete the settings as below:
 <br/>
-### Launch a cloud deployment
 
-The next step is to [launch a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. This can only be done through the spatial CLI.
+|Setting name|The setting you add|
+|-----|------|
+|**Assembly Name**| The name you gave the assembly in step 2 - for example; `myassembly`. <br/>(The name needs to conform to the following regex: `[a-zA-Z0-9_.-]{5,64}`.)|
+|**Deployment Name**| A name of your choice - for example; `shootyshooty`. <br/>(The name needs to conform to the following regex: `[a-z0-9_]{2,32}`.)|
+|**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
+|**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
```

Comment:
Project should not be capitalized. saw at least two instances of this in this doc